src/components/ModalPayment/ModalConfirmation/ModalConfirmation.tsx

```diff
@@ -46,13 +46,13 @@ const ModalConfirmation = (props: Props) => {
   const confirmationText = (purchaseType, sellerName, amount) => {
     switch (purchaseType) {
       case ModalPaymentTypes.modalPages.donation:
-        return `We appreciate your support. We'll let you know when ${sellerName} receives your donation!`;
+        return `We appreciate your support for ${sellerName}! You will receive an an email receipt shortly with the details of your donation.`;
```

Comment:
@nanxiy does Abby have a GitHub account to confirm these changes? She might be best at this lol

Reply:
@FelicityZhang you have an extra “an” in the message

Reply:
ahh sorry I was copy-pasta-ing

Reply:
> @nanxiy does Abby have a GitHub account to confirm these changes? She might be best at this lol

great idea, invited her !!

Reply:
Bump
```suggestion
        return `We appreciate your support for ${sellerName}! You will receive an email receipt shortly with the details of your donation.`;
```